pkg/plugins/backendplugin/manager/plugin_settings.go

```diff
@@ -20,6 +24,13 @@ func (ps pluginSettings) ToEnv(prefix string, hostEnv []string) []string {
 	return env
 }
 
+func checkForOverrides(key string, v string) string {
+	if value := os.Getenv(key); value != "" {
+		v = value
+	}
```

Comment:
Maybe would be good to have a logger here to print that certain config variables are overriden?